pkg/cmd/pipelinerun/describe.go

```diff
@@ -64,6 +71,25 @@ or
 				return err
 			}
 
+			if !opts.Fzf {
+				if _, ok := os.LookupEnv("TKN_USE_FZF"); ok {
+					opts.Fzf = true
+				}
+			}
+
+			if len(args) == 0 {
+				err = askPipelineRunName(opts, p)
+				if err != nil {
+					return nil
+				}
+				if output != "" {
+					pipelineRunGroupResource := schema.GroupVersionResource{Group: "tekton.dev", Resource: "pipelineruns"}
+					return actions.PrintObject(pipelineRunGroupResource, opts.PipelineRunName, cmd.OutOrStdout(), p, f, p.Namespace())
+				}
+
+				return prdesc.PrintPipelineRunDescription(s, opts.PipelineRunName, p)
```

Comment:
There is no need to repeat this. This is already happening outside the `if len(args) == 0 { .. }`. I would suggest finding a way to capture the PipelineRun name selected and use it in the logic below:

```go
if output != "" {
    pipelineRunGroupResource := schema.GroupVersionResource{Group: "tekton.dev", Resource: "pipelineruns"}
    return actions.PrintObject(pipelineRunGroupResource, args[0], cmd.OutOrStdout(), p, f, p.Namespace())
}

return prdesc.PrintPipelineRunDescription(s, args[0], p)
```

Reply:
This should still be removed. The point of the `if len(args) == 0 { ... }` block is to select the appropriate PipelineRun name. All the other logic within the `if` block is already happening. This should be fixed for all commands.